src/games/tictactoe/main.tsx

```diff
@@ -19,16 +18,6 @@ const TicTacToe = Client({
 });
 
 export default function() {
-    function lulz() {
```

Comment:
best code is node code :)

Reply:
I think my code snuck out of my local repo and made it to remote :)

I'm just guessing because I have used `lulz` often for codes that were not meant to be committed.